source/deploy/mattermost-changelog.md

```diff
@@ -11,6 +11,110 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
 - **Cloud Releases Prior to v9.2**: See the [Mattermost Legacy Cloud Changelog](https://docs.mattermost.com/deploy/legacy-cloud-changelog.html) for details.
 ```
 
+## Release v9.8 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: 2024-05-16**
+
+### Compatibility
+ - Updated minimum required Edge and Chrome versions to 122+.
+
+```{Important}
+If you upgrade from a release earlier than v9.7, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Upgraded Nodejs to v20.11.
```

Comment:
This is not UI related.

Reply:
I moved it to the Administration section.